.circleci/config.yml

```diff
@@ -69,6 +72,14 @@ jobs:
       - image: circleci/node:12
     steps:
       - checkout
+      - run:
+          name: Sync with latest master branch (only on PRs)
+          command: |
+            # if [ -n "$CIRCLE_PULL_REQUEST" ]
+            # then
+            #   git fetch origin refs/$CIRCLE_BRANCH/merge
```

Comment:
🤔, from the CircleCI document [Builtin Environment Variables](https://circleci.com/docs/2.0/env-vars/?gclid=CjwKCAiAzanuBRAZEiwA5yf4utDULIqLsdnr437DHKlQo30KqzhOa5Nxk7DS6ipsCaUO_ovC8V9osRoCD4YQAvD_BwE#built-in-environment-variables), shouldn't it be
```
git fetch origin refs/pull/$CIRCLE_PR_NUMBER/merge
```

Reply:
circle branch was = pull/1234. I can swap to the other

Reply:
That's okay if it works. Although it is a bit confused here since `pull/1234` is technically _not_ "the name of the Git branch currently being built", which should be something like `test262-pr-workflow`.

Reply:
I know what you mean.. Jenkins does the same thing here. weird